sdk/python/kfp/_client.py

```diff
@@ -307,12 +307,12 @@ def get_kfp_healthz(self):
     while not response:
       count += 1
       if count > max_attempts:
-        raise TimeoutError('API call timeout')
+        raise TimeoutError('Failed getting healthz endpoint after {} attempts.'.format(max_attempts))
       try:
         response = self._healthz_api.get_healthz()
         return response
-      except:
-        logging.info('Failed to get healthz info attempt {} of 5.'.format(count))
+      except Exception:
+        logging.exception('Failed to get healthz info attempt {} of 5.'.format(count))
```

Comment:
The default logging print level is warning. That's why info doesn't show up. 
I think we should probably change the default to INFO because I don't think we log too much from SDK.

That said, I think this change is still reasonable.

Reply:
I find it hard to change default to INFO, because we are using default logger for `logging` module, that's a global variable users also use.

Reply:
Yes, that's fine.